plexpy/activity_handler.py

```diff
@@ -187,11 +194,11 @@ def process(self):
                                              view_offset=self.timeline['viewOffset'])
                     # Start our state checks
                     if this_state != last_state:
-                        if this_state == 'paused':
+                        if this_state == 'paused' and progress_percent < 99:
```

Comment:
You shouldn't change it here, these `on_` functions do more than notifications. Especially the `stopped` state which triggers all the processing.